volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -2545,7 +2566,8 @@ def pageflags_enum(self) -> Dict:
 
         return pageflags_enum
 
-    @functools.cached_property
+    @property
+    @functools.lru_cache
```

Comment:
This method doesn't need to be a property at all, it only uses self to find the kernel, if anything this should be a LinuxUtility that takes in a kernel, or live somewhere else, there's no benefit to it living here.